### ISSUE TITLE
Support inviting other people to Zulip

### ISSUE THREAD

**Issue by gnprice** [zulip/zulip-mobile#3962]
It'd be good for the mobile app to provide the feature of inviting other people to your Zulip organization. Here's the existing user docs for the feature, as found in the webapp (and desktop app):
https://zulipchat.com/help/invite-new-users

Recording this now because a user [asked about it](https://chat.zulip.org/#narrow/stream/48-mobile/topic/invites/near/829521) today in chat. We haven't to date considered this a priority; it's generally an infrequent thing to need to do, and people have been content to use the webapp or desktop app for it. But filing this issue gives us a place to track if we start hearing repeated requests from more people.

**Comment by chrisbobbe** [comment]
> We haven't to date considered this a priority; it's generally an infrequent thing to need to do, and people have been content to use the webapp or desktop app for it.

We've noted at https://github.com/zulip/zulip-mobile/issues/3132 that onboarding is

> an area where we're at high risk of not noticing an issue ourselves or hearing reports from users, because developers and enthusiastic users don't go through these flows in our own normal usage.

— so it's good to have this issue open. 🙂

**Comment by gnprice** [comment]
That is true! I think this isn't quite that kind of onboarding issue, though -- maybe "onboarding" isn't even the right label for it, but it's the closest we have. In particular, an enthusiastic user may well regularly want to invite people to their Zulip org, even though they don't (and can't :slightly_smiling_face:) regularly use Zulip for the first time.
